src/components/themed/FioRequestRow.tsx

```diff
@@ -197,7 +198,9 @@ const emptyDenomination = { name: '', multiplier: '0' }
 export const FioRequestRow = connect<StateProps, {}, OwnProps>(
   (state, ownProps) => {
     const { fioRequest } = ownProps
-    const { defaultFiat, defaultIsoFiat } = state.ui.settings
+    const defaultIsoFiat = useSelector(state => state.ui.settings.defaultIsoFiat)
+    const defaultFiat = removeIsoPrefix(defaultIsoFiat)
```

Comment:
Get rid of this `defaultFiat`, and just pass `defaultIsoFiat` to `getFiatSymbol`

Reply:
Got it